sdk/typescript/src/utils/index.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { formatAddress, formatDigest } from './format.js';
+import {
+	isValidSuiAddress,
+	isValidSuiObjectId,
+	isValidTransactionDigest,
+	normalizeStructTag,
+	normalizeSuiAddress,
+	normalizeSuiObjectId,
+	parseStructTag,
+	SUI_ADDRESS_LENGTH,
+} from './sui-types.js';
+
+export { fromB64, toB64 } from '@mysten/bcs';
+export { is, assert } from 'superstruct';
```

Comment:
I'm fine with these being here now but we really need to deprecate these exports, as we're planning on removing the concrete superstruct exports.